src/main/scala/org/openeo/opensearch/OpenSearchResponses.scala

```diff
@@ -692,6 +692,20 @@ object OpenSearchResponses {
       decode[FeatureCollection](json)
         .valueOr(e => throw new IllegalArgumentException(s"${e.show} while parsing '$json'", e))
     }
+
+    private def retainTileIdPattern(features: Array[Feature], tileIdPattern: Option[String]): Array[Feature] =
+      tileIdPattern match {
+        case Some(pattern) => features.filter(feature => feature.tileID match {
+          case Some(tileId) =>
+            val matchesPattern = tileId matches pattern.replace("*", ".*")
+            logger.debug(s"${if (matchesPattern) "retaining" else "omitting"} feature with tileId $tileId")
+            matchesPattern
+          case _ =>
+            logger.debug(s"omitting feature with unknown tileId")
```

Comment:
Ok to add more info here? `logger.info(s"Omitting feature with unknown tileId: " + feature.id)`

Reply:
Sure, but the value of `feature.id` is surprising:

> omitting feature /eodata/Sentinel-2/MSI/L2A/2023/09/24/S2B_MSIL2A_20230924T103659_N0509_R008_T31UFS_20230924T132849.SAFE with tileId 31UFS

Reply:
A bit long id indeed. But might be handy in the future